db/models/sub_criteria.py

```diff
@@ -0,0 +1,66 @@
+import uuid
+
+from db import db
+
+
+class SubCriteria(db.Model):
+    sub_criteria_id = db.Column(
+        "sub_criteria_id",
+        db.Text(),
+        default=lambda: str(uuid.uuid4()),
```

Comment:
can this be type uuid as oppose to a string?

Reply:
great spot! have removed the str(), uuid has an appropriate __repr__ 👍